content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -96,41 +96,31 @@ Requires `dd-trace-php` version 0.71+.
 - **Socket I/O** shows the time taken waiting for a network read/write operation to execute.
-- **Object wait** shows the time waiting for a notify call on an object.
-- **Other** shows the time taken to execute the span that cannot be explained by profiling data.
+- **Monitor enter** shows the time a thread is blocked on a lock.
+- **Uncategorized** shows the time taken to execute the span that cannot be placed into one of the above categories.
 
-Click on one of these types to see a corresponding list, ordered by time, of the methods that are taking time. Clicking on the plus `+` will expand the stack trace to that method **in reverse order**.
-
-#### What does time spent in the 'Other' category mean?
-
-It is not uncommon to have a small amount of **Other** unexplained time (less than 10%). Potential reasons for Other time include:
-
-  - The span you selected isn't directly mapped to any execution. Profiling data is associated uniquely to spans when they are executing on a specific thread. For example, some spans are created and used uniquely as virtual containers of a series of related processing steps and never directly associated with any thread execution.
-  - Your application process cannot access CPU resources to execute and is paused. There is no way for the profiler to know about competing resources from other processes or containers.
-  - The application is locked in synchronization or in I/O events that are individually lower than 10ms: the Java profiler receives data for paused thread events (locks, I/O, parks) that are larger than 10ms. If you want to reduce that threshold, see [the documentation for changing setup defaults][1].
-  - The span you selected is short. Profiling is a sampling mechanism that regularly looks at how your code behaves. There might not be enough representative data for spans shorter than 50ms
-  - Missing instrumentation: Profiling breakdown requires that spans are associated with executing threads by activating these spans in the ScopeManager. Some custom instrumentations don't activate these spans properly, so you can't map them to executing threads. If this span comes from a custom integration, see the [Custom Instrumentation docs][2] for information on how to improve this.
+Clicking on the plus `+` expands the stack trace to that method **in reverse order**. Hovering over the value shows what percentage of time is explained by which category.
 
 ### Viewing a profile from a trace
 
 {{< img src="profiler/flamegraph_view.mp4" alt="Opening a view of the profile in a flame graph" video=true >}}
 
-For each type from the breakdown, click **View profile** to view the same data as what is shown in the flame graph.
-Click the **Span/Trace/Full profile** selector to define the scope of the data:
+For each type from the breakdown, click **View In Full Page** to view the same data in the full page view. From there you can change visualization to the flame graph.
```

Comment:
Trying to reword it so that we don't keep repeating view full page, but might not be as effective. 

```suggestion
For each type from the breakdown, click **View In Full Page** to see the same data opened up in a in a new page . From there you can change visualization to the flame graph.
```